test/verify/check-multi-machine-key

```diff
@@ -230,6 +230,7 @@ class TestMultiMachineKeyAuth(MachineCase):
                                     ".*: bridge program failed: Child process exited with code .*",
                                     # Since there is not password,
                                     # reauthorize doesn't work on m2
+                                    ".*: received authorize command for wrong user: user",
```

Comment:
Thanks. Two pull requests affecting the same code. Nice that we caught the failure early.